portfolio/src/main/webapp/comment.html

```diff
@@ -24,12 +24,18 @@ <h1>Comments</h1>
       <input type="submit" />
     </form>
 
-    <form action="/text" method="POST">
+    <!-- <form action="/text" method="POST">
       <p>Number of Comments You Would Like To Display:</p>
       <input type="number" name="player-choice" min="1" max="10" onchange="loadTasks()">
       <br/><br/>
-    </form>
-    <button id = 'delete' onclick="deleteComments()">Delete</button>
+    </form> -->
+    <!-- <button id = 'delete' onclick="deleteComments()">Delete</button> -->
```

Comment:
I think in general it's best practice to delete commented out code if you aren't using it -- I got this comment on one of my pull requests before :)

Reply:
Thank you for the tip I keep forgetting sometimes!